pkg/validate/raw/validate/namespace_validator.go

```diff
@@ -38,7 +38,7 @@ func validateNamespace(obj ast.FileObject) status.Error {
 	if !isValidNamespace(obj.GetName()) {
 		return nonhierarchical.InvalidNamespaceError(obj)
 	}
-	if configmanagement.IsControllerNamespace(obj.GetName()) {
+	if configmanagement.IsConfigSyncManagedNamespace(obj.GetName()) {
 		return nonhierarchical.IllegalNamespace(obj)
```

Comment:
`IllegalNamespace` also needs to be updated to reflect the namespace.

Reply:
Done in the function.